communities/src/tests.rs

```diff
@@ -198,71 +198,6 @@ fn new_community_works() {
     });
 }
 
-#[test]
```

Comment:
why remove this test?

Reply:
the bucket size was the same resolution as the used geohash.. I do, though not remember whether this was with the final resolution of the geohash. As I understood the test, it verified whether it is possible to register two communities with the same parameters except for location (which deviates slightly). Since the new version would take into account such small differences in location parameter, the test itself would be contradictory to the change.